tests/storage/test_mount.py

```diff
@@ -0,0 +1,43 @@
+import threading
+import subprocess
+import time
+import sys
+sys.path.append("..")
+import _thread as thread
+import os
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from test_main import Test
+import argparse
+
+class Mount(Test):
+    """
+    Checks for eos mount points
+    """
+    def __init__(self):
+        params = {}
+        params['test_name'] = "mount"
+        Test.__init__(self, **params)
```

Comment:
Use `parent()` instead of `Test`. (same for all the other classes, ofc)
Check the documentation on how to call parent constructors in Python3.

Reply:
Maybe I should use super() here?

Reply:
That's what I wanted to say before my brain freezed :)